AntWars/GamePanel.cs

```diff
@@ -5,16 +5,23 @@
 using AntWars.Board;
 using AntWars.Board.Ants;
 using AntWars.Helper;
-using System.Drawing;
 
 namespace AntWars {
 
     partial class GamePanel : Form {
 
-        private const string Player1Carry = "ff008000";
-        private const string Player2Carry = "ff0000ff";
-        private const string Player1Scout = "ff2e8b57";
-        private const string Player2Scout = "ff6a5acd";
+        private static readonly Color COLOR_PLAYER1_CARRY = Color.Olive;
+        private static readonly Color COLOR_PLAYER1_SCOUT = Color.Green;
+        private static readonly Color COLOR_PLAYER1_WARRIOR = Color.Lime;
+        private static readonly Color COLOR_PLAYER1_BASE = Color.YellowGreen;
+
+        private static readonly Color COLOR_PLAYER2_CARRY = Color.DarkCyan;
+        private static readonly Color COLOR_PLAYER2_SCOUT = Color.Blue;
+        private static readonly Color COLOR_PLAYER2_WARRIOR = Color.DeepSkyBlue;
+        private static readonly Color COLOR_PLAYER2_BASE = Color.BlueViolet;
```

Comment:
@jmattheis War ursprünglich auch mein Plan, welcher aufgrund von [Fehlermeldungen](http://stackoverflow.com/questions/10340396/why-is-it-not-possible-to-declare-a-constant-of-type-system-drawing-color) nicht aufging.